src/otk_external_osbuild/command.py

```diff
@@ -46,7 +46,7 @@ def source_add_curl(data, checksum, url):
 
 def depsolve_dnf4():
     data = json.loads(sys.stdin.read())
-    tree = data["tree"]
+    tree = data["tree"]["otk.external.osbuild_depsolve_dnf4"]
```

Comment:
Dol we need a similar change for the two other externals we have (file_from_text and file_from_path)?

Reply:
Yes we do. Will be in followup under #88.